### ISSUE TITLE
Can you add pipe style menus?

### ISSUE THREAD

**Issue by mrgreenlinux** [joewing/jwm#450]
Looking at documentation I was thinking of adding a pipe style menu, to simply display text. Not sure if Dynamic is the right way to go?

**Comment by technosaurus** [comment]
Can you give an example?

**Comment by mrgreenlinux** [comment]
https://www.techrepublic.com/blog/linux-and-open-source/four-great-pipe-menus-for-openbox/

I am trying to simply display information (text) via root menu. ie

`<tag>Alt+t Open Terminal</tag>`

**Comment by Israel-D** [comment]
@mrgreenlinux Yes, you can use the <Dynamic> tag for this.
JWM and openbox menus are similar (they are XML) so you could, potentially with some work, even convert some of those 'pipe' menus.
Any program that prints out JWM xml menus (binary, scripts, etc..) can be used as a dynamic menu.
see this [link](http://www.joewing.net/projects/jwm/config-2.3.shtml#root) for more info.
tl;dr
```xml
<JWM>
  <RootMenu dynamic="exec:pipemenu1.sh" onroot="a">
  <RootMenu>
    <Program label="terminal">xterm</Program>
    <Dynamic label="cool menu">exec:pipemenu2</Dynamic>
  </RootMenu>
<!-- the rest of your .jwmrc -->
</JWM>
```
Just make sure your dynamic menus output only valid JWM menus i.e.
```bash
#!/bin/sh
printf "%s" "<JWM>
<Program label=\"item 1\">xterm</Program>
<Menu label=\"submenu\">
  <Program label=\"item 2\">geany</Program>
</Menu>
</JWM>
"
exit 0
```

**Comment by mrgreenlinux** [comment]
It looks like it would work, just need text output... will have to test if program tag can be empty. Openbox had a item tab that simply displays text. Or maybe add dev/null (empty script) "thinking out loud"

Thanks so much for your help, really enjoying JWM so many features and options to play with ...

**Comment by Israel-D** [comment]
@mrgreenlinux Have you resolved this issue yet?
